tests/rptest/services/openmessaging_benchmark_worker.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import re
+import threading
+import os
+
+from ducktape.services.service import Service
+from ducktape.utils.util import wait_until
+
+
+# worker need to be started near redpanda process #ToDo: fix comment
+class OpenMessaging_Benchmark_Worker():
+    PERSISTENT_ROOT = "/var/lib/openmessaging"
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "worker.log")
+    PORT = 9090
+    STATS_PORT = 9091
+
+    def __init__(self, redpanda):
+        self.redpanda = redpanda
+        for node in self.redpanda.nodes:
+            self.start_node(node)
+
+    def start_node(self, node):
+        self.redpanda.logger.error(
+            "Starting openmessaging benchmark worker node on %s",
+            node.account.hostname)
+
+        node.account.ssh("mkdir -p %s" %
+                         (OpenMessaging_Benchmark_Worker.PERSISTENT_ROOT))
+
+        start_cmd = "cd /opt/openmessaging-benchmark; bin/benchmark-worker --port %s --stats-port %s >> %s 2>&1 & disown" % \
+                (OpenMessaging_Benchmark_Worker.PORT,
+                 OpenMessaging_Benchmark_Worker.STATS_PORT,
+                 OpenMessaging_Benchmark_Worker.STDOUT_STDERR_CAPTURE)
+
+        with node.account.monitor_log(OpenMessaging_Benchmark_Worker.
+                                      STDOUT_STDERR_CAPTURE) as monitor:
+            node.account.ssh(start_cmd)
+            monitor.wait_until("Javalin has started",
+                               timeout_sec=100,
+                               backoff_sec=7,
+                               err_msg=" service didn't finish startup")
+        self.redpanda.logger.error(
+            "OpenMessaging_Benchmark_Worker is successfully started on node %s.",
+            node.account.hostname)
+
+    def stop(self):
+        for node in self.redpanda.nodes:
+            self.stop_node(node)
+
+    def stop_node(self, node):
+        self.redpanda.logger.error(
+            "Stopping %s node on %s" %
+            (type(self).__name__, node.account.hostname))
+        node.account.kill_process("benchmark", allow_fail=False)
+
+
+# worker need to be started near redpanda process #ToDo: fix comment
+class OpenMessaging_Benchmark(Service):
+    PERSISTENT_ROOT = "/var/lib/openmessaging"
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "becnhmark.log")
+    OPENMESSAGING_DIR = "/opt/openmessaging-benchmark"
+    DRIVER_FILE = "driver-redpanda/redpanda-ducktape.yaml"
+    WORKLOAD_FILE = "workloads/ducktape.yaml"
+
+    def __init__(self, ctx, redpanda):
+        self.redpanda = redpanda
+        self.set_default_configuration()
+        super(OpenMessaging_Benchmark, self).__init__(ctx, num_nodes=1)
```

Comment:
if the super class doesn't depend on customizations, it is customary to initialize it first thing